Source/WebCore/platform/graphics/displaylists/DisplayListItems.cpp

```diff
@@ -280,9 +280,9 @@ NO_RETURN_DUE_TO_ASSERT void DrawFilteredImageBuffer::apply(GraphicsContext&) co
     ASSERT_NOT_REACHED();
 }
 
-void DrawFilteredImageBuffer::apply(GraphicsContext& context, ImageBuffer* sourceImage, FilterResults& results)
+void DrawFilteredImageBuffer::apply(GraphicsContext& context, RefPtr<ImageBuffer> sourceImage, FilterResults& results)
```

Comment:
This is a weird change. As far as I know, we're not supposed to change parameter types during this smart pointer effort, unless there is a good reason to. Here we're just calling `.get()` on the smart pointer so no good reason I can see.

Reply:
Ah yes, I was told about that, but that patch was already in-flight so I forgot to revert this, will do...